android/src/com/mapswithme/maps/MwmActivity.java

```diff
@@ -1115,7 +1117,7 @@ private boolean removeFragment(String className, boolean animate)
   }
 
   // Called from JNI.
-  @Override
+  @Override @Keep
```

Comment:
Could you please keep annotations on new lines to follow existing convention in the code?